patchwork/step.py

```diff
@@ -130,6 +130,11 @@ def debug(self, inputs):
         MAX_LENGTH = 1000 # Max limit to print inputs
         printed_chars = 0
         for key, value in inputs.items():
+            if "api_key" in key.lower():
+                if len(value) >= 24:
+                    value = f"{value[:4]}{'*' * (len(value) - 8)}{value[-4:]}"
+                else:
+                    value = "*" * len(value)
```

Comment:
To be honest we can just return a constant string and disassociate any information about the value of the key from the censor/mask. I recommend just either `******` or `<masked api key>`.

Reply:
Okay seems like returning ```<masked api key>``` sounds better, Shall I go ahead with that then?

Reply:
Yes, please do. Thank you.